packages/okta-vue/package.json

```diff
@@ -8,18 +8,19 @@
     "src"
   ],
   "scripts": {
+    "build": "rimraf dist/ && cross-env NODE_ENV=production webpack --config webpack.config.js --output-library-target=umd -p",
+    "build:harness": "yarn --cwd test/e2e/harness install",
+    "build:package-info": "node ../../util/write-package-info.js . src/packageInfo.js",
+    "kill:port": "kill -9 $(lsof -t -i:8080 -sTCP:LISTEN) || true",
```

Comment:
TODO: We're closing the port (if open) **before** calling our tests. This means that there is a chance the dev server is still running once they finish.